Dockerfile

```diff
@@ -101,6 +101,8 @@ FROM gcr.io/distroless/static as workflow-controller
 
 USER 8737
 
+WORKDIR /home/argo
+
```

Comment:
do we really need this? can't we use absolute paths for files in `persist/sqldb/sqldb.go`

Reply:
WorkDir is needed to refer to where all the binary and files need to be copied. Can we config the path?